requirements.txt

```diff
@@ -11,5 +11,6 @@ pylint>=2.15.10
 pytest-cov>=4.0.0
 pytest-random-order>=1.1.0
 pytest>=7.2.0
+rich==13.7.1
```

Comment:
Do we have permission to use this library?

Reply:
yes